### PR TITLE
Fix URL checks/remove unmaintained projects

### PR DESCRIPTION
**Pull request by nodiscc**
Fixes part of https://travis-ci.org/github/awesome-selfhosted/awesome-selfhosted

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -854,7 +850,6 @@ _Games, game servers and control panels._
 **[`^        back to top        ^`](#)**
 
 - [BlueMind](https://www.bluemind.net/en/) - Groupware with email, calendar, addressbooks, exchange active sync, exchange "mapi" protocol support. ([Source Code](https://forge.bluemind.net/stash/projects/BM/repos/bluemind-public/browse)) `AGPL-3.0` `Java`
-- [Citadel](http://www.citadel.org/doku.php) - Groupware including email, calendar/scheduling, address books, forums, mailing lists, IM, wiki and blog engines, RSS aggregation and more. ([Source Code](http://www.citadel.org/doku.php/installation:source)) `GPL-3.0` `C`
```
Comment (n8225) [inline README.md:857]: Citadel appears to still be maintained http://easyinstall.citadel.org/manifest.html

File: README.md [hunk 2]
```diff
@@ -1347,8 +1341,7 @@ _See also [Ticketing](#ticketing), [Task management/To-do lists](#task-managemen
 - [Re:Backlogs](https://github.com/kaishuu0123/rebacklogs) - Project management and collaboration tool powered by Ruby on Rails & VueJS. ([Demo](https://rebacklogs.saino.me/users/sign_up)) `MIT` `Ruby`
 - [Redmine](http://www.redmine.org/) - Redmine is a flexible project management web application. ([Demo](http://demo.redmine.org/), [Source Code](https://svn.redmine.org/redmine/)) `GPL-2.0` `Ruby`
 - [RhodeCode](https://rhodecode.com/) - RhodeCode is an open source platform for software development teams. It unifies and simplifies repository management for Git, Subversion, and Mercurial. ([Demo](https://try.rhodecode.com/), [Source Code](https://code.rhodecode.com/)) `AGPL-3.0` `Python`
-- [SCM Manager](https://www.scm-manager.org/) - The easiest way to share and manage your Git, Mercurial and Subversion repositories over http. ([Source Code](https://bitbucket.org/sdorra/scm-manager/src)) `BSD-3-Clause` `Java`
-- [sorcia](https://sorcia.org) - Sorcia is a minimalist web frontend for Git repositories with fine-grained user access controls, multiple themes to choose based on your taste and many more. ([Source Code](https://git.mysticmode.org/r/sorcia)) `AGPL-3.0` `Go`
```
Comment (n8225) [inline README.md:1351]: Here is a mirror of Sorcia https://github.com/mysticmode/sorcia last commit was a little over a month ago.  Not sure if we want to hold off on removing this.

File: README.md (hunk 1 above)
Comment (nodiscc) [reply README.md:857]: dropped the commit

File: README.md (hunk 2 above)
Comment (nodiscc) [reply README.md:1351]: Website is still down after two months, no recent commits, no activity on mailing lists. I think it is safer to remove Sorcia and wait for someone to re-add it - in the current state it looks abandoned.

**Pull request merged**